tests/Gaufrette/Functional/Adapter/AzureMultiContainerBlobStorageTest.php

```diff
@@ -182,14 +200,28 @@ public function shouldKeepFileObjectInRegister()
      */
     public function shouldWrtieToSameFile()
     {
-        $FileObjectA = $this->filesystem->createFile('container11/somefile');
+        $path = $this->getContainerName('container') . '/somefile';
+
+        $FileObjectA = $this->filesystem->createFile($path);
         $FileObjectA->setContent('ABC');
 
-        $FileObjectB = $this->filesystem->createFile('container11/somefile');
+        $FileObjectB = $this->filesystem->createFile($path);
         $FileObjectB->setContent('DEF');
 
         $this->assertEquals('DEF', $FileObjectB->getContent());
+    }
+
+    private function getContainerName($prefix)
```

Comment:
It's not obivous this method won't return the same result when called with the same args. I'd prefer something like `createUniqueContainerName($prefix)`